src/commands/math.ts

```diff
@@ -307,19 +331,20 @@ var MathCommand = P(MathElement, function(_, super_) {
         } while (nesting > 0);
       }
     }
-    return tokens.join('').replace(/>&(\d+)/g, function($0, $1) {
-      return ' mathquill-block-id=' + blocks[$1].id + ' aria-hidden="true">' + blocks[$1].join('html');
+    return tokens.join('').replace(/>&(\d+)/g, function(_$0:string, $1:string) {
+      var num1 = parseInt($1, 10);
+      return ' mathquill-block-id=' + blocks[num1].id + ' aria-hidden="true">' + blocks[num1].join('html');
     });
   };
 
   // methods to export a string representation of the math tree
-  _.latex = function() {
-    return this.foldChildren(this.ctrlSeq, function(latex, child) {
+  latex () {
+    return this.foldChildren(this.ctrlSeq || '', function(latex, child) {
```

Comment:
ctrlSeq || ''   appears often enough throughout this changeset that we probably want to consider making ctrlSeq accessed by a function/method that does this coalescing rather than requiring so many references spread around to remember to add this.

Reply:
I started writing a ticket for this, but now I'm not 100% convinced that it's actually a good idea. I see this idiom show up 5 times in the codebase in functions that build either a latex or text or mathspeak representation of a node.

We could put this pattern behind an accessor, but it's not totally obvious to me that treating an undefined ctrlSeq identically to an empty string ctrlSeq would be the right thing to do in every case.